test/unit/test.speech_to_text.v1.js

```diff
@@ -223,6 +223,27 @@ describe('speech_to_text', function() {
     });
   });
 
+  describe('recognizeWebM()', function() {
+    const session_path = '/v1/sessions/foo/recognize';
+    const payload = {
+      audio: fs.createReadStream(__dirname + '/../resources/sample1.webm'),
+      content_type: 'audio/webm'
+    };
+
+    it('should generate a valid payload with session', function() {
+      const req = speech_to_text.recognize(extend({ session_id: 'foo' }, payload), noop);
+      assert.equal(req.uri.href, service.url + session_path);
+      assert.equal(req.method, 'POST');
+      assert.equal(req.headers['Content-Type'], payload.content_type);
+      assert.equal(req.src.path, payload.audio.path);
+    });
+
+    it('should check first four hexs for webm file', function() {
+      const file = fs.readFileSync(__dirname + '/../resources/sample1.webm');
+      assert.equal(file.slice(0, 4).toString(), '\u001aEߣ');
```

Comment:
This test, while not being a bad idea, doesn't actually exercise our code - it's just checking your resource file. I'd say keep it, but change the description to something like "the sample webm file should have the expected header"

And, then, like I was saying above, there should be another test that does exercise our content-type detection code. Perhaps two, one that goes through the whole process of creating a request, and one that just calls `RecognizeStream.getContentType(fs.readFileSync(__dirname + '/../resources/sample1.webm'));`